src/gui/rss/automatedrssdownloader.cpp

```diff
@@ -599,9 +597,11 @@ void AutomatedRssDownloader::updateFieldsToolTips(bool regex)
               + "<li>" + tr("Whitespaces count as AND operators (all words, any order)") + "</li>"
               + "<li>" + tr("| is used as OR operator") + "</li></ul></p>"
               + "<p>" + tr("If word order is important use * instead of whitespace.") + "</p>";
-        ui->lineContains->setToolTip(tip);
-        ui->lineNotContains->setToolTip(tip);
     }
+
+    tip += "<p>" + tr("An expression with an empty ") + "<tt>|</tt>" + tr(" clause (e.g. ") + "<tt>" + tr("expr|") + "</tt>" + tr(")");
```

Comment:
Not exactly sure what you're saying here - did you put stuff in the code block you didn't mean to? I'll push a new version and you tell me if I've done what you wanted or not.